users/service.go

```diff
@@ -53,7 +53,7 @@ type Service interface {
 	ViewProfile(ctx context.Context, token string) (User, error)
 
 	// ListUsers retrieves users list for a valid admin token.
-	ListUsers(ctx context.Context, token string, offset, limit uint64, email string, meta Metadata) (UserPage, error)
+	ListUsers(ctx context.Context, token string, state string, offset, limit uint64, email string, meta Metadata) (UserPage, error)
```

Comment:
`state string, offset, limit uint64, email string, meta Metadat` should be passed as a `PageMetadata` structure